tests/test_isrTaskLSST.py

```diff
@@ -2037,6 +2037,99 @@ def test_bssVoltageChecks(self):
                 deferredChargeCalib=self.cti,
             )
 
+    def test_hvBiasChecks(self):
+        """Test the HVBIAS checks."""
+        # We use a flat frame for this test for convenience.
+        mock_config = self.get_mock_config_no_signal()
+        mock_config.doAddDark = True
+        mock_config.doAddFlat = True
+        # The doAddSky option adds the equivalent of flat-field flux.
+        mock_config.doAddSky = True
+
+        mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        input_exp = mock.run()
+
+        isr_config = self.get_isr_config_electronic_corrections()
+        isr_config.doBias = True
+        isr_config.doDark = True
+        isr_config.doFlat = False
+        isr_config.doDefect = True
+
+        # Set the voltage.
+        input_exp.metadata["HVBIAS"] = "OFF"
+
+        # Check that processing runs with checks turned off.
+        isr_config.doCheckUnprocessableData = False
+
+        isr_task = IsrTaskLSST(config=isr_config)
+        with self.assertNoLogs(level=logging.WARNING):
+            _ = isr_task.run(
+                input_exp.clone(),
+                bias=self.bias,
+                dark=self.dark,
+                crosstalk=self.crosstalk,
+                ptc=self.ptc,
+                linearizer=self.linearizer,
+                defects=self.defects,
+                deferredChargeCalib=self.cti,
+            )
+
+        # Check that processing runs with other way of turning checks off.
+        isr_config.doCheckUnprocessableData = True
+        isr_config.bssVoltageMinimum = 0.0
```

Comment:
Isn't the voltage for that mock 50 V? In that case, the voltage is always greater than bssVoltageMinimum wetheir it is 0 or 10 as set here, or 5, the default.

Reply:
This part of the test is to set this config to 0.0 which turns off all voltage checking (including HVBIAS).   I could add a `doCheckVoltage` but it seemed redundant.